packages/block-library/src/button/richTextWrapper.native.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalUseGradient,
+} from '@wordpress/block-editor';
+/**
+ * Internal dependencies
+ */
+import styles from './editor.scss';
+
+function RichTextWrapper( { children, borderRadiusValue, backgroundColor } ) {
```

Comment:
Any reasons we call this RichTextWrapper? As far as I see it is only for applying gradient and there's no rich text usage here. We can just call it GradientBackground? or just LinearGradient?

Reply:
It was my _working name_ for that component and agree with you that it's not cool. Generally, that component is a container for `RichText` which uses `LinearGradient` once it's defined otherwise regular `View` with a specified background color. My suggestion is to call it `background-container`, wdyt?

Reply:
since it is all about colors I think I'd go with `color-background`

Reply:
Changed name to `color-background`